Src/DependencyCollector/Net46.Tests/DependencyTrackingTelemetryModuleTestNet46.cs

```diff
@@ -108,7 +120,7 @@ public void TestDependencyCollectionEventSource()
 
         [TestMethod]
         [Timeout(5000)]
-        public void TestDependencyCollectionEventSource404()
+        public async Task TestDependencyCollectionEventSource404()
```

Comment:
I think 404 is not even a failure, it's non 20* status code, and fail could mean that request was not send at all (e.g. because of DNS or network issue). Let's call in as NonSuccessStatusCode